odd-platform-ui/src/components/DataEntityDetails/Lineage/Lineage.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import AppGraphContainer from 'components/shared/AppGraph/AppGraphContainer';
 import { DataEntityLineageById } from 'redux/interfaces/dataentityLineage';
-import { StylesType } from './LineageStyles';
+import { Container } from './LineageStyles';
 
-interface LineageProps extends StylesType {
+interface LineageProps {
   isLoaded: boolean;
   dataEntityId: number;
   lineage: DataEntityLineageById;
 }
 
-const Lineage: React.FC<LineageProps> = ({ classes, dataEntityId }) => (
-  <div className={classes.container}>
+const Lineage: React.FC<LineageProps> = ({ dataEntityId }) => (
+  <Container container sx={{ mt: 2 }}>
```

Comment:
Looks like a Container styled component is only used in one place, so why do we set an inline style here and not in LineageStyles.ts?

Reply:
Moved inline styles to LineageStyles.ts